go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/crc-org/machine-driver-libvirt
 
 require (
-	github.com/crc-org/machine v0.0.0-20221028075518-f9b43442196b
+	github.com/crc-org/machine v0.0.0-20240926103419-a943b47fd48b
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.8.4
+	github.com/stretchr/testify v1.9.0
 	libvirt.org/go/libvirt v1.8000.0
 	libvirt.org/go/libvirtxml v1.9008.0
```

Comment:
libvirtxml is native go code, so we can update it if we want, it's only `libvirt` which links to a particular libvirt version. Fine with me to keep this unchanged.

Reply:
As of now let's keep this unchanged.